### PR TITLE
make playground win32 use comctl v6

### PR DESCRIPTION
**Pull request by asklar**
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/react-native-windows/pull/8062)

### CODE REVIEW

**Review by chiaramooney — APPROVED** [review]
👍

File: packages/playground/windows/playground-win32/Application.manifest [hunk 1]
```diff
@@ -18,6 +18,19 @@
     type="winb32"
     version="1.0.0.0" />
 
+  <dependency>
+    <dependentAssembly>
+      <assemblyIdentity
+          type="win32"
+          name="Microsoft.Windows.Common-Controls"
+          version="6.0.0.0"
```
Comment (jonthysell) [inline packages/playground/windows/playground-win32/Application.manifest:26]: Why is this necessary?

Comment (asklar) [reply packages/playground/windows/playground-win32/Application.manifest:26]: by default apps using comctl get comctl 5.82 version (XP style) which uses older controls.

Comment (jonthysell) [reply packages/playground/windows/playground-win32/Application.manifest:26]: ok, so no risk of a win10 machine with a version < 6.0?

Comment (asklar) [reply packages/playground/windows/playground-win32/Application.manifest:26]: correct, this should be available on Vista+

**Review by jonthysell — APPROVED** [review]

**Review by dannyvv — APPROVED** [review]
